### PR TITLE
How to: guide for Linux users

### PR DESCRIPTION
**Pull request by boisterousbandit**
Detailed steps to run FOFIX. Users can copy and paste lines into Terminal.

### CODE REVIEW

**Review by Linkid — COMMENTED** [review]
This is really great to have this doc. But did you see this one : https://fofix.readthedocs.io/en/latest/quickstart/installation.html ?
I think it could be a good idea to mix your debian / ubuntu installation guide (you should make add this detail) with this existing doc ;). In addition, this installation guide will be in the `doc` directory then.

**Review by Linkid — CHANGES_REQUESTED** [review]
In addition, as I said in IRC, I think you should write your commit message in small case (like a description), not in upper case (it is a detail, I can fix it for you or tell you how to fix it if you need help).

**Comment by boisterousbandit** [comment]
Thank you for your thorough review. I have made the changes you requested. Please let me know if there are other changes I need to make.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ This is Frets on Fire X, a highly customizable rhythm game supporting many modes
 
 Setup
 -----
+[How-To-Guide For Linux Users] (/doc/Linux_Setup_Guide.md)
```
Comment (Linkid) [inline README.md:19]: A space between the text and the link is breaking the rendering.
About the file, it should be in `doc/source/quickstart/`.

File: doc/Linux_Setup_Guide.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+1. Open *Terminal*
+2. Enter: `cd ~/Documents/fofix-master`
+3. Enter: `python setup.py msgfmt`
+
+####Step 13: Install Requirements
+1. Open *Terminal*
+2. Enter: `cd ~/Documents/fofix-master`
+3. Enter: `pip install -r requirements.txt`
+
+####Step 14: Run FOFIX
```
Comment (Linkid) [inline doc/Linux_Setup_Guide.md:92]: Case: FoFiX ;)

**Review by Linkid — CHANGES_REQUESTED** [review]
Great :). Some mistakes to fix and it should be good.
About the file, it should be in `doc/source/quickstart/`. And you didn't link this new doc to the existing one, btw (don't know if it is on purpose or not).

File: README.md (hunk 1 above)
Comment (boisterousbandit) [reply README.md]: (not intentional at all). Good catch. Fixed the location.

File: doc/Linux_Setup_Guide.md (hunk 2 above)
Comment (boisterousbandit) [reply doc/Linux_Setup_Guide.md]: Fixed.

**Review by Linkid — APPROVED** [review]

**Pull request merged**